man/systemd-sbsign.xml

```diff
@@ -85,11 +99,44 @@
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--signed-attributes=<replaceable>PATH</replaceable></option></term>
+        <term><option>--signed-attributes-signature=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>Configure the signed attributes blob (as provided by
+        <command>export-signed-attributes</command>) and corresponding signature for the
+        <command>sign</command> command.</para>
```

Comment:
this is in pkcs7 format, no? should be mentioned

Reply:
maybe also say that this is input for `sign`, not `output`

i mean, that's what this is, right?

i take it this implements the preparation of the offline signing, and then accepting the results, but not the offline signing itself (as that is done on some vendor host), right, hence would be good to indicate that

Reply:
It's not in pkcs7 format actually, it's just the DER encoded signed attributes that have to be signed.

Reply:
yeah, but those attributes follow pkcs7 semantics, no?